mysite/myapp2/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from .models import Client, Product, Order
+
+class ClientForm(forms.ModelForm):
+    class Meta:
+        model = Client
+        fields = ['name', 'email', 'phone_number', 'address']
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'price', 'quantity']
+
+class OrderForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        fields = ['client', 'products', 'total_amount', 'order_date']
```

Comment:
Поле `order_date` не редактируемое
```suggestion
        fields = ['client', 'products', 'total_amount']
```